docs/hooks.md

```diff
@@ -51,4 +46,38 @@ Ark v0.7.0+ continues to support the original (deprecated) way to specify pre ho
 Please see the documentation on the [Backup API Type][1] for how to specify hooks in the Backup
 spec.
 
+## Hook Example with fsfreeze
+
+We are going to walk through using both pre and post hooks for freezing a file system. Freezing the
+file system is useful to ensure that all pending disk I/O operations have completed prior to taking a snapshot.
+
+We will be using [example/nginx-app/with-pv.yaml][2] for this example. Follow the [steps for your provider][3] to
+setup this example.
+
+### Annotations
+
+The Ark [example/nginx-app/with-pv.yaml][2] serves as an example of adding the pre and post hook annotations directly
+to your declarative deployment. Below is an example of what updating an object in place might look like.
+
+```shell
+kubectl annotate pod -n nginx-example -l app=nginx \
+    pre.hook.backup.ark.heptio.com/command='["/sbin/fsfreeze", "--freeze", "/var/log/nginx"]' \
+    pre.hook.backup.ark.heptio.com/container=fsfreeze \
+    post.hook.backup.ark.heptio.com/command='["/sbin/fsfreeze", "--unfreeze", "/var/log/nginx"]' \
+    post.hook.backup.ark.heptio.com/container=fsfreeze
+```
+
+Now test the pre and post hooks by creating a backup. You can use the Ark logs to verify that the pre and post
+hooks are running and exiting without error.
+
+```shell
+ark backup create nginx-hook-test
+
+ark backup get nginx-hook-test
+ark backup logs nginx-hook-test | grep hookCommand
+```
+
+
 [1]: api-types/backup.md
+[2]: https://github.com/heptio/ark/blob/master/examples/nginx-app/with-pv.yaml
```

Comment:
just use `examples/nginx-app/with-pv.yaml` -- makes things work properly with GH pages/multiple versions of the docs